js/mainjs/config.js

```diff
@@ -33,7 +33,10 @@ function configManager(path) {
 	 */
 	config.attr = function(key, value) {
 		if (value !== undefined) {
-			this[key] = value;
+			config[key] = value;
+		}
+		if (config[key] === undefined) {
+			config[key] = null;
```

Comment:
This is for the line in `addIPCListeners.js` because a significant error (it throws a popup instead of just logging stuff) occurs:
```js
event.returnValue = response ? response : null;
```
`event.returnValue` apparently can't be `undefined`